css/styles.css

```diff
@@ -204,6 +218,15 @@ nav {
   background-color: #1a2236;
 }
 
+.projectNameX {
+  font-family: PoppinsLight, sans-serif;
+  font-size: 20px;
+  color: white;
+  padding-left: 25px;
+  margin-top: 20px;
```

Comment:
You have put great effort into your project. Kindly work on the horizontal scroll of the page in order to improve the user experience.

 
![image](https://user-images.githubusercontent.com/20554583/200152360-d4da784d-dcc0-4c4b-97bd-4ebeb32f63ea.png)